week-1_review/Task1

```diff
@@ -0,0 +1,219 @@
+create table customer(
+	customer_id serial primary key,
+	name varchar(50),
+	email varchar(50) unique not null,
+	created_at timestamp default now()
+);
+
+create table product(
+	product_id serial primary key,
+	name varchar(50) not null,
+	category varchar(50),
+	price numeric(10,2) check (price>0)
+);
+
+create table orders (
+    order_id serial primary key,
+    customer_id int references customer(customer_id),
+    product_id int references product(product_id),
+    quantity int CHECK (quantity > 0),
+    order_date timestamp default NOW()
+);
+
+
+insert into customer(name, email)
+values 
+('Muhammad Ali', 'mali@gmail.com'),
+('Abbas Zaidi', 'abbas@outlook.com'),
+('Ahmed Raza', 'ahmed.raza@gmail.com'),
+('Sana Tariq', 'sana.tariq@gmail.com'),
+('Bilal Hussain', 'bilal.hussain@gmail.com'),
+('Zainab Ali', 'zainab.ali@gmail.com'),
+('Hassan Iqbal', 'hassan.iqbal@gmail.com'),
+('Fatima Noor', 'fatima.noor@gmail.com'),
+('Usman Khalid', 'usman.khalid@gmail.com'),
+('Laiba Sheikh', 'laiba.sheikh@gmail.com');
+
+insert into product(name, category, price)
+values
+('Laptop', 'electronics', 98000.00),
+('smartphone', 'electronics', 50000.00),
+('desk chair', 'furniture', 5500.00),
+('water bottle', 'kitchen', 80.00),
+('notebook', 'stationery', 250.50);
+
+
+insert into orders(customer_id, product_id, quantity)
+values
+(1, 1, 2),
+(1, 2, 1),
+(1, 3, 1),      
+(2, 3, 1),
+(2, 4, 3),
+(3, 5, 5),
+(3, 1, 1),
+(3, 2, 2),   
+(4, 2, 2),
+(4, 3, 1),
+(5, 4, 2),
+(5, 5, 3),
+(6, 1, 1),
+(6, 2, 1),
+(7, 3, 2),
+(7, 4, 1),
+(8, 5, 2),
+(8, 1, 3),
+(8, 4, 1),      
+(9, 2, 1),
+(9, 3, 1),
+(10, 4, 2),
+(10, 5, 1);
+
+--List all customers who ordered more than 2 different products.
+select customer_id, count(distinct product_id) as product_ordered
+from orders
+group by customer_id
+having count(distinct product_id) > 2;
```

Comment:
It would be better to show customer details, not only ID. A customer is identifiable to everyone by his/her name.